pages/en/lb3/Attaching-models-to-data-sources.md

```diff
@@ -55,9 +55,26 @@ The tool will then add an entry such as the following to `datasources.json`:
 
 This example creates a MongoDB data source called "mongo1". The identifier determines the name by which you refer to the data source and can be any string.
 
+Next, the generator will prompt you to type the datasource settings, such as
+host, port, user, password and database name, as well as to install the database connector.
+
+```shell
+? Enter the datasource name: mongo1
+? Select the connector for mongodb: MongoDB (supported by StrongLoop)
+? Connection String url to override other settings (eg: mongodb://username:password@hostname:port/database):
+? host: your-mongodb-server.foo.com
+? port: 27017
+? user: demo
+? password: ****
+? database: demo
+? Install loopback-connector-mongodb@^1.4 Yes
+```
+
+If you choose not to type the database credentials, and/or `no` for installing the database connector, then you can install it manually and edit `server/datasources.json` as in the following steps.
```

Comment:
it is an actual `and/or` situation because the user may choose to do both manually or one of the two conditions